app/helpers/min-admins.js

```diff
@@ -0,0 +1,29 @@
+import Ember from 'ember';
+
+/**
+ * minAdmins helper - used to determine if the user should be able to update permissions
+ * of a particular contributor.  False if user is trying to update the permissions
+ * of the sole admin.  All projects need at least one administrator.
+ *
```

Comment:
Thanks for adding documentation! But note that a few more fields are needed for this to work with YUIDoc, Documenting the parameters and return types are useful, but excluding the method name causes it not to appear in auto-generated docs at all.

(YUIDoc doesn't really try to understand code; it just strings together docs based on docstrings.)